src/common/adapters/AxelarAdapter.sol

```diff
@@ -41,15 +41,15 @@ contract AxelarAdapter is Auth, IAxelarAdapter {
     //----------------------------------------------------------------------------------------------
 
     /// @inheritdoc IAxelarAdapter
-    function file(bytes32 what, string calldata axelarId, uint16 centrifugeId, string calldata source) external auth {
+    function file(bytes32 what, string calldata axelarId, uint16 centrifugeId, string calldata source) public auth {
```

Comment:
This can be `external` again

Reply:
Ah sorry for missing that!

Reply:
[6fff57f](https://github.com/centrifuge/protocol-v3/pull/458/commits/6fff57f0e695d4684dd24387dfe15461c2fdd490)